.gitignore

```diff
@@ -1,3 +1,9 @@
 node_modules
 npm-debug.log
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+## generated files
+ng-inspector.firefox/*.xpi
+ng-inspector.firefox/data/icons/
+ng-inspector.firefox/data/ng-inspector.js
```

Comment:
@DrewML no problem from my side to remove these paths from the gitignore and add the generated assets in the pull request once reviewed

I prefer to keep the generated files out of the repo (just to prevent any chance to have the committed versions to be out of sync with their last generated version) but in this case I preferred to do not add them into the branch just to keep it clean to help the review to focus only on the effective differences (and add them later in a separate commit in this branch).